### PR TITLE
Updated to Chart.js v3.0.0-beta.3

### PR DESCRIPTION
**Pull request by santam85**
Thanks for contributing.

- [ V ] Tick to sign-off your agreement to the [Developer Certificate of Origin (DCO) 1.1](https://developercertificate.org)

## Description

Updated code to function correctly when using latest chart.js 3.0.0-beta.3

## Testing

Tested including the updated library in ng2-charts: https://github.com/valor-software/ng2-charts/pull/1276

### CODE REVIEW

**Comment by benmccann** [comment]
I'm on my phone so can't quite tell, but it looks like you may have changed the tabs indentation to spaces. If so could you please change it back?

File: src/controller.financial.js [hunk 1]
```diff
@@ -50,10 +49,10 @@ Chart.defaults.financial = {
 
 				let val = DateTime.fromMillis(ticks[0].value);
 				if ((majorUnit === 'minute' && val.second === 0)
-						|| (majorUnit === 'hour' && val.minute === 0)
-						|| (majorUnit === 'day' && val.hour === 9)
-						|| (majorUnit === 'month' && val.day <= 3 && val.weekday === 1)
-						|| (majorUnit === 'year' && val.month === 1)) {
+          || (majorUnit === 'hour' && val.minute === 0)
```
Comment (benmccann) [inline src/controller.financial.js:52]: spaces -> tabs

File: src/controller.financial.js [hunk 2]
```diff
@@ -143,10 +142,10 @@ class FinancialController extends Chart.controllers.bar {
 	}
 
 	/**
-	 * Implement this ourselves since it doesn't handle high and low values
-	 * https://github.com/chartjs/Chart.js/issues/7328
-	 * @protected
-	 */
+   * Implement this ourselves since it doesn't handle high and low values
```
Comment (benmccann) [inline src/controller.financial.js:145]: spaces -> tabs

File: src/controller.financial.js [hunk 3]
```diff
@@ -189,8 +188,8 @@ class FinancialController extends Chart.controllers.bar {
 	}
 
 	/**
-	 * @protected
-	 */
+   * @protected
```
Comment (benmccann) [inline src/controller.financial.js:191]: spaces -> tabs

**Comment by santam85** [comment]
@benmccann would you be open to release this as a beta/with the `next` tag? It would simplify testing in preparation to ChartJS 3.0 on the other projects and for users.

**Pull request merged**